docs/_articles/en/v3.x/getting-started/install.md

```diff
@@ -60,5 +60,15 @@ TOPICS
 #### Upgrade plug-in
 Automatic steps don't apply here since this is a pilot version. Follow steps from above to uninstall and install directly again.
 
+Note: By default, the first time you execute a ```v3``` command, the ```<HOME_DIR>/.sfdx-scanner/Config.json``` from your
```

Comment:
Can we continue using the terminology "2.x" and "3.x" to keep the contents consistent?

Reply:
Good point. Stand by for that change.